lib/msf/base/sessions/unix_escaping.rb

```diff
@@ -0,0 +1,29 @@
+module Msf::Sessions
+  module UnixEscaping
+    def shell_command_token(cmd,timeout = 10)
+      shell_command_token_unix(cmd,timeout)
+    end
+
+    # Convert the executable and argument array to a command that can be run in this command shell
+    # @param cmd_and_args [Array<String>] The process path and the arguments to the process
+    def to_cmd(cmd_and_args)
+      escaped = cmd_and_args.map do |arg|
+        escape_arg(arg)
+      end

```

Comment:
```suggestion
      escaped = cmd_and_args.map { |arg| escape_arg(arg) }
```

I think this syntax is a bit more idiomatic